src/ng-select/ng-dropdown-panel.component.ts

```diff
@@ -210,6 +210,11 @@ export class NgDropdownPanelComponent implements OnInit, OnChanges, OnDestroy, A
             return;
         }
 
+        if ($event.target && $event.target.shadowRoot && $event.composedPath() && $event.composedPath()[0]
```

Comment:
there is no need for additional if, just add variable with path or composedPath and keep old check
```js
const path = $event.path || ($event.composedPath && $event.composedPath())
```

Reply:
Made the change that was requested! :)